src/js/client/controls.js

```diff
@@ -0,0 +1,150 @@
+// Present a standard set of user controls with login/logout/profile links
+// TODO: Convert this to use a common "toggler" grade once that has been sufficiently exercised and moved somewhere central.
+/* global fluid, jQuery */
+(function ($) {
+    "use strict";
+    var gpii = fluid.registerNamespace("gpii");
+    fluid.registerNamespace("gpii.express.user.frontend.controls");
+
+    gpii.express.user.frontend.controls.handleMenuKeys = function (that, event) {
```

Comment:
Consider the use of the Infusion Keyboard Accessibility plugin "activatable" in order to collect this information in a more easily visible declarative form: 
https://wiki.fluidproject.org/display/docs/Keyboard+Accessibility+Plugin+API
(docs are old, but still current).
We have an old tutorial here:
https://wiki.fluidproject.org/display/Infusion14/Tutorial+-+Keyboard+Accessibility+Plugin
Ideally, the arguments to the activatable plugin will just be captured in a set of mixin grade definition options.